app/common/locale/content.json

```diff
@@ -2571,21 +2617,39 @@
     "selectCase": {
       "title": "Dewiswch achos",
       "heading": "Eich apeliadau budd-daliadau",
+      "ibca-heading": "Eich apeliadau",
       "appealReference": "Cyfeirnod yr apêl",
       "benefit": "Budd-dal",
+      "ibca-benefit": "Apêl",
       "dateSubmitted": "Dyddiad cyflwyno",
       "mrnDate": "Dyddiad MRN",
+      "ibca-mrnDate": "Hysbysiad o Benderfyniad Adolygiad",
       "action": "Cam gweithredu",
       "view": "Gweld"
     },
     "assignCase": {
       "title": "Nodwch y cod post ar gyfer yr apêl",
-      "message": "Hwn yw’r cod post a ddefnyddiwyd pan gafodd yr apêl ei chofrestru",
+      "appealTypeOptions": {
+        "ibca": {
+          "radioOption": "Apêl Iawndal Gwaed Heintiedig",
+          "inputText": "Rhowch y cyfeirnod IBCA a ddarperir yn eich gohebiaeth Hysbysiad o Benderfyniad Adolygiad neu unrhyw ohebiaeth gan yr IBCA."
+        },
+        "otherBenefits": {
+          "radioOption": "Apêl Budd-dal",
+          "inputText": "Nodwch y cod post ar gyfer yr apêl",
+          "hintText": "Dyma’r cod post a ddefnyddiwyd pan gofrestrwyd yr apêl"
+        },
       "errors": {
-        "noPostcode": "Nid ydych wedi nodi cod post. Nodwch eich cod post cyn parhau.",
-        "invalidPostcode": "Nid yw’r manylion rydych wedi’u rhoi yn cyd-fynd â’r wybodaeth sydd gennym, cysylltwch â ni os gwelwch yn dda",
-        "tyaNotProvided": "Please use the MYA link in the most recent email we've sent you",
-        "postcodeDoesNotMatch": "Nid yw’r manylion rydych wedi’u rhoi yn cyd-fynd â’r wybodaeth sydd gennym, cysylltwch â ni os gwelwch yn dda"
+        "missing": {
+          "appealType": "Nid ydych wedi dewis math o apêl. Dewiswch eich math o apêl cyn parhau.",
+          "postcode": "Nid ydych wedi nodi cod post. Nodwch eich cod post cyn parhau.",
+          "ibcaReference": "Nid ydych wedi rhoi cyfeirnod IBCA. Rhowch eich cyfeirnod IBCA cyn parhau.",
+          "tya": "Defnyddiwch y ddolen MYA yn yr e-bost ddiweddaraf y bu inni ei anfon atoch chi"
+        },
+        "invalid": {
+          "postcode": "Nid yw’r manylion rydych wedi’u rhoi yn cyd-fynd â’r wybodaeth sydd gennym, cysylltwch â ni os gwelwch yn dda",
+          "ibcaReference": "Nid yw’r cyfeirnod IBCA a roddwyd gennych wedi’i adnabod. Gwiriwch y cyfeirnod a rhowch gynnig arall arni."
+        }
```

Comment:
remove this close curly bracket

Reply:
Taleb helped to identify the right bracket to move, which was in evidence